quickstarts/ansible-automation-controller/config.yml

```diff
@@ -0,0 +1,45 @@
+
+
+slug: ansible-automation-controller
+description: |
+  ## Why monitor Ansible Automation Controller?
+  An ideal Ansible Automation Controller performance monitor must offer comprehensive and actionable information that software developers need to troubleshoot an application successfully. Some key components are:
+
+  - Track the efficiency of business transactions starting using the Ansible Automation Controller server.
+  - Use deep diagnostics to analyze and fix issues with application performance.
+  - Drill down to the Ansible Automation Controller and the associated application components.
+  - Detects faults to reduce application downtime.
+  - Compare the performance of your Ansible Automation Controller servers.
+  
+  ## Comprehensive monitoring quickstart for Ansible Automation Controller
+  
+  New Relic's Ansible Automation Controller monitoring quickstart boasts instant full-stack observability out-of-the-box:
```

Comment:
When you monitor your Ansible Automation Controller server with New Relic, you get all of this full-stack observability out of the box:

Reply:
Hi @**paperclypse** ,
Reviewed text correction has been changed. Kindly review it.